data/docs/example.mdx

```diff
@@ -123,3 +104,40 @@ section.Content {
   padding: 1rem;
 }
 ```
+
+## styles.ts ✨
+
+The styles file is where the magic happens! It contains all of the CSS Components that are imported into the main React component.
+
+As you can see, CSS Components are nothing more than a way to map CSS classes to React components.
+
+You can either create this file yourself, [or use our helpful CLI tool](/docs/cli).
+
+```ts
+import { styled } from "@phantomstudios/css-components";
+
+import css from "./styles.module.css";
+
+export const PanelWrapper = styled("div", {
+  css: css.PanelWrapper,
+  variants: {
+    wide: {
+      true: css.PanelWrapper_wide,
```

Comment:
```suggestion
      true: css.PanelWrapper_Wide,
```

Variants should be capitalised

Reply:
Not per the docs https://www.css-components.net/docs/cli

Worth double checking

Reply:
I think we should update the docs to match in a seperate PR then, the projects css components has been used on follow the Component_VariantName convention

Reply:
When using the CLI command to generate styles.ts,  the Component_VariantName convention would generate capitalised props (e.g. `<Panel Wide />`). I assume that's why it used lowercase in the first place? What do you think?

Reply:
Possibly? In practice, we've never used the CLI tool, and have likely developed our convention ignoring that existed. IMO, the CLI tool can probably be safely deprecated, as we've like, never used it.

I think we should fit our style guide to our real world usage, and not to technical constraints of a part of the tool that no one used.

Reply:
https://github.com/phantomstudios/css-components-website/issues/50